apps/prairielearn/src/ee/auth/lti13/lti13Auth.ts

```diff
@@ -245,6 +261,12 @@ async function launchFlow(req: Request, res: Response, next: NextFunction) {
 
   const parameters = OIDCLaunchFlowSchema.passthrough().parse({ ...req.body, ...req.query });
 
+  // Generate our own OIDC state, use it to toggle if testing is happening
+  let state = crypto.randomBytes(28).toString('hex');
+  if (parameters?.RelayState === 'test') {
```

Comment:
Rather than use `RelayState` (which is a SAML-specific concept and is only used because SAML allows for SP-initiated logins), could we instead use `?test` or `?test=true`?

Reply:
Changed to `?test` in 517b8a1